<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arco</class>
 <widget class="QDialog" name="Arco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>440</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>440</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>440</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>420</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>420</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>420</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Radio:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Angulo:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Angulo:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
